tools/ci/ci_wptrunner_infrastructure.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
 set -ex
 
-SCRIPT_DIR=$(cd $(dirname "$0") && pwd -P)
+REL_DIR_NAME=$(dirname "$0")
+SCRIPT_DIR=$(cd "$REL_DIR_NAME" && pwd -P)
 WPT_ROOT=$SCRIPT_DIR/../..
-cd $WPT_ROOT
+cd "$WPT_ROOT"
 
 run_infra_test() {
-    TERM=dumb ./wpt run --log-mach - --yes --manifest ~/meta/MANIFEST.json --metadata infrastructure/metadata/ --install-fonts --install-webdriver --log-wptreport="/home/test/artifacts/wptreport-$1.json" $2 $1 infrastructure/
+    ./tools/ci/taskcluster-run.py "$1" "$2" -- --metadata=infrastructure/metadata/ --log-wptreport="../artifacts/wptreport-$1.json" --include=infrastructure/
```

Comment:
This makes this file specific to TC; I know we mostly put Azure-specific stuff in tools/ci/azure, but tools/ci/tc is more our TC support code, so maybe we need a new directory? Would be good to make move it somewhere TC specific.

(Currently we only invoke `taskcluster-run.py` from within `tools/ci/tc/tasks/test.yml`, as far as I can tell.)

Reply:
So, fun fact, I have a followup that makes these scripts explicitly TC-specific (which I think in practice they all are), but I'd like to land this first.